### ISSUE TITLE
`\DescribeOption` adds "(env.)" suffix to options

### ISSUE THREAD

**Issue by gusbrs** [latex3/latex3#1111]
As the title says, the current version of `l3doc` appends "(env.)" to options rendered with `\DescribeOption`.

MWE:

```tex
\documentclass{l3doc}

\begin{document}

\DescribeOption{option}

\end{document}
```
Produces:

![Screenshot from 2022-07-06 17-47-07](https://user-images.githubusercontent.com/19410606/177640158-3c4767ec-ec9c-4643-a513-243effea2cdc.png)

**Comment by FrankMittelbach** [comment]
l3doc is really just a hack and we don't have much time to improve on it. It is only available  *as is*. Anyway, it contains a manual definition that misuses the Env support from doc V2 to provide further support  but that fails now as environments identify themselves by default.

You can try this for now:
```
\documentclass{l3doc}

\usepackage{lipsum}

\NewDocElement[printtype = \textit{option}]{Option}{optionenv}  

\begin{document}

\DescribeOption{option} bla bla

\begin{optionenv}{myoption}
\lipsum[1][1-3]
\begin{verbatim}
   code for myoption
\end{verbatim}
\end{optionenv}

\end{document}
```

I used `optionenv` because option as an environment name might be a little dangerous, but of course you can choose any name you like

**Comment by gusbrs** [comment]
Thank you for the prompt response and the workaround.

> l3doc is really just a hack and we don't have much time to improve on it. It is only available *as is*.

This was just a report, not a complaint. I've read the "Use at own risk!" warnings in the documentation. I just presumed you'd like to know if anyone had identified a misbehavior regardless of the status of the class.

**Comment by FrankMittelbach** [comment]
> This was just a report, not a complaint. I've read the "Use at own risk!" warnings in the documentation. I just presumed you'd like to know if anyone had identified a misbehavior regardless of the status of the class.

I haven't taken it as such, and yes I do appreciate having such issues logged (and eventually resolved). I was just recording why it has a (somewhat) lower priority

**Comment by josephwright** [comment]
@FrankMittelbach  I was just looking at this as it affects e.g. `siunitx`. You suggested fix puts 'option' after all of the `\DecribeOption` usage, which may or may not be desirable (feels a bit wordy to me). However, in the index I end up with `(optionenv)` after each entry: that's true even if I remove `printtype = \textit{option}`. I'm wondering if that is a bug in `doc`, and even if not, if there's a way to control the index text, ideally independently of what's used in the main part.

**Comment by FrankMittelbach** [comment]
The defaults for printing in the margin or in the index are set up to be like the envname so you have to do more than "removing it", you have to empty it.
```
\NewDocElement[printtype = \textit{option},idxtype=option,idxgroup=options]
  {Option}{optionenv}  

%\RenewDocElement[printtype = ,idxtype=, idxgroup=options]
%  {Option}{optionenv}  
```
both work with different results
